README.md

```diff
@@ -1,40 +1,34 @@
-App for [publishers.brave.com](https://publishers.brave.com).
-
 [![Build Status](https://travis-ci.org/brave-intl/publishers.svg?branch=master)](https://travis-ci.org/brave-intl/publishers)
 
-## Quick start
-
-### Setup
+## :wrench: Setup
 
-These steps presume you are using OSX and [Homebrew](https://brew.sh/).
+Follow these steps to setup the App for [publishers.brave.com](https://publishers.brave.com). This guide presumes you are using OSX and [Homebrew](https://brew.sh/).
```

Comment:
Url should be https://publishers.basicattentiontoken.org/

Reply:
I was so curious about this, it seems to redirect but regardless I'll fix.